changelogs/fragments/add-os-server-tag.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
please remove this changelog fragment, this is going to be auto generated for new modues.

Reply:
Done